test/scenarios.py

```diff
@@ -5419,7 +5419,7 @@
                             0, FILES["testfile-small"].id, Error.FILE_REJECTED
                         ),
                     ),
-                    action.SleepMs(200),
```

Comment:
minor: I think `SleepMs()` is a bit redundant, `Sleep()` is enough to be had in the codebase, especially when we can pass `0.2` instead of `200`